docker-compose.yaml

```diff
@@ -1,18 +1,21 @@
+version: '3'
 services:
   reverse-proxy:
-    image: traefik:2.5
+    image: traefik:latest
```

Comment:
I am a little afraid to bind to latest for containers which we have no control over. I remember, for example, breaking changes from mongo on major releases. On the contrary, I agree we should update the version here

Reply:
I think for traefik it is fine

Reply:
but what's the advantage in the end? If this is only to limit the required manual changes, why don't we set snyk or similar going forward which will propose major upgrades? In this way, we can always rollback and have no unnoticed upgrades

Reply:
Yes, that would be an option. 

But I fear a bit that people without  too much knowlegde will also  skip regularly  updating their git checkout to get up-to-date and then still run with an outdated traefik (there  really frequent updates  to expect, but breaking changes should actually be rare).

And we should make it really clear, that scicatlive is fine for experimenting, but should not be used unchanged for any production workload.

Reply:
We have the option that we fix to the latest release now, but we make sure that we update scicat live when there is a new release of the containers. However, I think it is unlikely we can prioritize this, so I would say fixing to the latest release is fine here.

Reply:
yes, I think that is coming as a separate PR, at least upgrading to the latest tag. Then I would suggest using snyc or similar